docker/mysql/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debezium/example-mysql:1.5
+
+COPY music_streaming.sql /docker-entrypoint-initdb.d/
```

Comment:
you could mount this file instead of writing a custom Dockerfile but it's fine too